Example/IntegrationTests/Sign/Helpers/ClientDelegate.swift

```diff
@@ -1,10 +1,8 @@
 import Foundation
 @testable import WalletConnectSign
+import Combine
 
-class ClientDelegate: SignClientDelegate {
-    func didChangeSocketConnectionStatus(_ status: SocketConnectionStatus) {
-        onConnected?()
-    }
+class ClientDelegate {
```

Comment:
actually when we do not depend on sign delegate anymore we could remove this class entirely.
maybe we can create an issue and do it at some point?

Reply:
ClientDelegate usage looks good in tests for me

Reply:
we could subscribe for SignClients events only now